src/index.html

```diff
@@ -13,7 +13,21 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <div id="qa-block" class="border first-block">
+    <div class="border second-block">
```

Comment:
use class names that descibe the element semantic, 
first, second, third etc. - not the right names